src/Reshuffle.ts

```diff
@@ -6,6 +6,25 @@ import { BaseConnector, BaseHttpConnector, EventConfiguration } from 'reshuffle-
 import { createLogger } from './Logger'
 import { Logger, LoggerOptions } from 'winston'
 import http from 'http'
+import { promises as fs } from 'fs'
```

Comment:
@zacmos @duckranger why are changes in Reshuffle a part of this PR? Seems like a merge problem - please do not include these?